install.sh

```diff
@@ -351,6 +369,43 @@ fetch() {
 	sh_c mv "$FILE.incomplete" "$FILE"
 }
 
+install_terraform() {
+	# Check if the unzip package is installed. If not error peacefully.
+	if !(command_exists unzip); then
+		echoh
+		echoerr "This script needs the unzip package to run."
+		echoerr "Please install unzip to use this function"
+		exit 1
+	fi
+	TERRAFORM_VERSION="1.3.4"
```

Comment:
Could we move this higher up so the URL above auto-includes this too?

Reply:
Yes, I will do that real quick.

Reply:
How do you think putting it on line 160 would feel?
```
main() {
	if [ "${TRACE-}" ]; then
		set -x
	fi
	TERRAFORM_VERSION="1.3.4"
	unset \
		DRY_RUN \
		METHOD \
		OPTIONAL \
		ALL_FLAGS \
		RSH_ARGS \
		EDGE \
		RSH

	ALL_FLAGS=""
	while [ "$#" -gt 0 ]; do
		case "$1" in
		-*)
			ALL_FLAGS="${ALL_FLAGS} $1"
			;;
		esac
...
```

Reply:
Will putting it above the `if` statement be good, or higher?

Reply:
Seems fine!

Reply:
Cool, I will add the commit real quick!